Readme.md

```diff
@@ -1,26 +1,47 @@
 Slack app for Voting on Stories and Tickets remotely and asynchronously.
 
+## Usage
+
+1. Install this app to your Slack team visiting http://asyncvoter.tk and pressing on the `Add to Slack` button.
```

Comment:
Isn't it possible to add this to an existing Slack team?
(How did it get added to AgileVentures?)